hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/service/NftAllowanceServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.restjava.service;
+
+import com.hedera.mirror.common.domain.entity.NftAllowance;
+import com.hedera.mirror.restjava.common.RangeOperator;
+import com.hedera.mirror.restjava.repository.NftAllowanceRepository;
+import jakarta.inject.Named;
+import java.util.Collection;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+@Named
+@RequiredArgsConstructor
+public class NftAllowanceServiceImpl implements NftAllowanceService {
+
+    public static final String TOKEN_ID = "token_id";
+    public static final String OWNER = "owner";
+    public static final String SPENDER = "spender";
+    private static final Sort OWNER_TOKEN_ASC_ORDER =
+            Sort.by(Sort.Direction.ASC, OWNER).and(Sort.by(Sort.Direction.ASC, TOKEN_ID));
```

Comment:
A helper method and static imports would clean it up.

```suggestion
    private static final Sort OWNER_TOKEN_ASC_ORDER = sort(ASC, OWNER, TOKEN_ID);
```